### PR TITLE
Add new rpc

### PR DESCRIPTION
**Pull request by xiaoluSF**
Feasibility test remaining.

### CODE REVIEW

**Comment by xiaoluSF** [comment]
`getVersion` not confirm yet.

File: Source/RPC/Structures/Version.swift [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import Foundation
+
+public struct Version: Decodable {
+    public var softwareVersion: String
+
+    enum CodingKeys: String, CodingKey {
+        case softwareVersion
+    }
+
+    public init(from decoder: Decoder) throws {
```
Comment (ashchan) [inline Source/RPC/Structures/Version.swift]: `Version` only contains a primitive `String` property. Do we really need to implement `init(from: Decoder)`?

Comment (xiaoluSF) [reply Source/RPC/Structures/Version.swift]: `{
    "jsonrpc":"2.0",
    "id":83,
    "result":{
        "softwareVersion":"v0.22.0"
    }
}`
This is the data structure, it's different from other data structure like
`
{
    "id": 83,
    "jsonrpc": "2.0",
    "result": "0x1d10"
}
`

**Comment by ashchan** [comment]
@xiaoluSF Does the test fail because the testnet we use hasn't upgraded to support this new RPC methods yet?

**Comment by xiaoluSF** [comment]
Yes.

**Comment by ashchan** [comment]
> Yes.

OK I see. Do you know when the public testnet would upgrade? Don't want to merge this without all tests passing.

**Comment by xiaoluSF** [comment]
> > Yes.
> 
> OK I see. Do you know when the public testnet would upgrade? Don't want to merge this without all tests passing.

May be a week after CITA released (May 7), too late.

**Review by ashchan — APPROVED** [review]

**Pull request merged**